fastlane/lib/fastlane/actions/git_add.rb

```diff
@@ -7,9 +7,9 @@ def self.run(params)
           UI.success("Successfully added from \"#{paths}\" 💾.")
         elsif params[:path]
```

Comment:
@ohayon Btw, since we are deprecating the `pathspec` parameter, shouldn't we not include it in the example code?